### DISCUSSION TITLE
Peter Griffin template

### DISCUSSION THREAD

**Discussion by JomSpoons** [machineonamission/mediaforge#34] [Ideas]
A Peter Griffin template based on the [Peter Griffin Explains the Joke](https://amp.knowyourmeme.com/memes/peter-griffin-explains-the-joke) meme would be a good addition to the captioning templates. It'd work similarly to the preexisting *eminem* and *stuff* commands.

![495](https://user-images.githubusercontent.com/42208891/128647141-f35c8447-f2d3-43e5-a40f-989f2383c1d0.jpg)

**Comment by machineonamission** [comment]
added in 3e4868980928dba9752b508963d17a45b333886c

![image](https://user-images.githubusercontent.com/35177464/128654133-0714daac-bfe9-47d0-bfe4-9c1c5dcb11d5.png)
![image](https://user-images.githubusercontent.com/35177464/128654191-6181cd0d-f5ae-45c6-bb4c-62e466457c14.png)
